src/job_shop_scheduler.py

```diff
@@ -3,24 +3,36 @@
 solve a Job Shop Scheduling problem using CQM.
 
 """
+
 from __future__ import annotations
 
 import argparse
 import sys
 import warnings
+from enum import Enum
 
 import pandas as pd
 from dimod import Binary, ConstrainedQuadraticModel, Integer
 from dwave.system import LeapHybridCQMSampler, LeapHybridNLSampler
 
 sys.path.append("./src")
-import utils.scipy_solver as scipy_solver
+from dwave.optimization.generators import flow_shop_scheduling
+
 import utils.plot_schedule as job_plotter
+import utils.scipy_solver as scipy_solver
 from model_data import JobShopData
 from utils.greedy import GreedyJobShop
 from utils.utils import print_cqm_stats, write_solution_to_file
 
-from dwave.optimization.generators import flow_shop_scheduling
+
+class SamplerType(Enum):
+    HYBRID = 0
+    HIGHS = 1
+
+
+class HybridSamplerType(Enum):
+    CQM = 0
+    NL = 1
```

Comment:
NL should be the default selected solver.
```suggestion
class HybridSamplerType(Enum):
    NL = 0
    CQM = 1
```